ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -411,8 +411,13 @@ echo "1. Review the generated configuration file:"
 echo "   cat ${OUTPUT_FILE}"
 echo ""
 echo "2. Move it to the appropriate directory:"
-echo "   mv ${OUTPUT_FILE} ci-operator/config/openshift/sandboxed-containers-operator/"
+echo "mv ${OUTPUT_FILE} ci-operator/config/openshift/sandboxed-containers-operator/"
 echo ""
-echo "3. Generate and update CI configuration before creating PR:"
-echo "   make ci-operator-config && make prow-config && make jobs && make update"
+echo "3. Add to git:"
+echo "git add ci-operator/config/openshift/sandboxed-containers-operator/${OUTPUT_FILE}"
+echo ""
+echo "4. Generate and update CI configuration before creating PR:"
+echo "make ci-operator-config && make registry-metadata && make prow-config && make jobs && make update
+echo ""
+echo "5. git add changes, commit and push to PR"
```

Comment:
Since you already added the changes in step 3 it shouldn't be mentioned here (simple push and open PR would do)